version/version.go

```diff
@@ -13,13 +13,17 @@ var (
 	// Whether cgo is enabled or not; set at build time
 	CgoEnabled bool
 
-	// The main version number that is being run at the moment.
-	Version = "1.8.5"
-	// A pre-release marker for the version. If this is "" (empty string)
+	// The next version number that will be released. This will be updated after every release
+	// Version must conform to the format expected by github.com/hashicorp/go-version
+	// for tests to work.
+	// A pre-release marker for the version can also be specified (e.g -dev). If this is omitted
 	// then it means that it is a final release. Otherwise, this is a pre-release
 	// such as "dev" (in development), "beta", "rc1", etc.
+
+	Version           = ""
```

Comment:
Is it possible to keep the version number specified in this file and just overwrite post release?

Are build tools and some of our testing expects the version information to be set. The failing test on this PR are checking for a minimum Packer version, which is not being set. Building this PR locally gives an incorrect Packer version. 

```
~>  packer version
Packer v0.0.0-dev
```


> One thing to note is that the version.go and base_version.go files are used for local development and will be now overwritten by the ld-flags with the actual version that are being called in the go builds.

I found this statement in the PR description does this mean we can modify this file for local development?

Reply:
@nywilken

It's fine to set in version.go as long as `build.yml` uses the new actions-set-product-version action (which uses version/VERSION and gets used [here](https://github.com/hashicorp/packer/pull/12135/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R44)) :) I've updated version.go to use the current Packer version `1.8.5`

Reply:
Thanks @claire-labry I'll update our release documentation to call out the versioning is now handled by CRT in version/VERSION but that for local development we should always keep version/version.go manually up to date.